appveyor.yml

```diff
@@ -19,8 +27,17 @@ deploy:
     server:
     api_key:
       secure: ejD7WLs3aqlXgHLyWq444QK6pnMT6g/QiRxptqIGaOgKvGM7SJ17r/QH6e1jEJ9M
+    skip_symbols: true
     on:
       branch: master
+  - provider: NuGet
+    artifact: /.*\.nupkg/
+    server: https://www.myget.org/F/easynetq-unstable/api/v2/package
+    api_key:
+       secure: <add_api_key>
```

Comment:
This isn't apparent that it needs to be changed. Possibly update description to the PR for them to update the API key.